pylint/checkers/variables.py

```diff
@@ -2329,6 +2339,45 @@ def _check_classdef_metaclasses(self, klass, parent_node):
 
         return consumed
 
+    def _node_should_be_skipped(
+        self, consumer: NamesConsumer, node: nodes.Name, is_start_index: bool
```

Comment:
```suggestion
    def _node_should_be_skipped(
        self, node: nodes.Name, consumer: NamesConsumer, is_start_index: bool
```
I would suggest to put `node` first. That is usually how it's done in the other helper methods.